openhtf/io/user_input.py

```diff
@@ -63,7 +63,8 @@ def __init__(self):
     self._response = None
     self._cond = threading.Condition()
 
-  def DisplayPrompt(self, message, text_input=False):
+  def DisplayPrompt(self, message, text_input=False,
+                    timeout=FLAGS.prompt_timeout_s):
```

Comment:
It's a good convention to include the units in the argument name, so 'timeout' should be 'timeout_s' if it's seconds.